sysutils/tmux-pasteboard/Portfile

```diff
@@ -15,8 +15,8 @@ long_description    ${description}
 
 depends_run         path:bin/tmux:tmux
 
-checksums           rmd160  647603906f23e3f1781c75b627c500d5f04b3391 \
-                    sha256  3472ff08a1250a20bfc955310bf5855175c60a109176bcefdd22ac517418af70
+checksums           rmd160  2cc454a397f3cc8b5b86a2140f85be97eee060e5 \
+                    sha256  e779a4a4f916484f15b9b384bbfa892030c70b65340da68f0d474df43889bcfd
```

Comment:
We now add a "size" field as well. ("port -v checksum" will give you the correct line to add.)

If you fix this, it should be possible to merge your change pretty fast. Be sure to squash your commits.

Reply:
That was more or less the right thing to do.